analyses/cell-type-ewings/scripts/utils/tumor-validation-helpers.R

```diff
@@ -122,6 +122,7 @@ create_marker_gene_df <- function(
 # For any genes that are in the specified `cell_type`, sum of the logcounts is calculated
 # output is a data frame with barcodes and `{cell_type}_sum`
 calculate_sum_markers <- function(marker_genes_df,
+                                  sce,
                                   type) {
```

Comment:
Just noting that I think `styler` might complain about this tab pattern, so you might want to run it here to save some future revisions.

Reply:
I have `styler` set up in my local pre commit file and I just ran it again just on this file and it didn't make any changes.